integration-tests/sanity/sanity_test.go

```diff
@@ -307,7 +307,7 @@ var _ = Describe("Sanity", func() {
 	Context("should  delete jiva volume", func() {
 		It("should delete jiva pvc", func() {
 			// Getting the PVC artifact
-			artifact, err := artifacts.GetArtifactUnstructured(artifacts.JivaPVCArtifacts)
+			artifact, err := artifacts.GetArtifactUnstructuredFromFile(artifacts.JivaPVCArtifacts)
```

Comment:
Mark a TODO as these things should not be in artifacts yaml.
Only the openebs operator contents will reside in artifacts yaml

Reply:
added